numpy/_core/src/umath/ufunc_object.c

```diff
@@ -5177,6 +5182,7 @@ ufunc_dealloc(PyUFuncObject *ufunc)
         Py_DECREF(ufunc->identity_value);
     }
     Py_XDECREF(ufunc->obj);
+    //Py_XDECREF(ufunc->dict);
```

Comment:
Don't forget to uncomment again!

Reply:
As initially discussed in the comment above, this `Py_XDECREF` is causing segfault on multiple jobs.

Reply:
Done!